libyul/backends/evm/EVMDialect.cpp

```diff
@@ -142,6 +142,23 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 			Expression const& arg = _call.arguments.front();
 			_assembly.appendLinkerSymbol(std::get<Literal>(arg).value.str());
 		}));
+
+		builtins.emplace(createFunction(
+			"memoryguard",
+			1,
+			1,
+			SideEffects{true, true, true, false, false},
+			{LiteralKind::Number},
+			[](
+				FunctionCall const& _call,
+				AbstractAssembly& _assembly,
+				BuiltinContext&,
+				function<void(Expression const&)> _visitExpression
+			) {
+				visitArguments(_assembly, _call, _visitExpression);
```

Comment:
So whatever the visitor leaves on the stack is kept on the stack, e.g. becomes the "return" value?

Reply:
Yes